misc-utils/lsfd.c

```diff
@@ -37,12 +37,32 @@
 #include <sys/uio.h>
 #include <linux/sched.h>
 #include <sys/syscall.h>
-#include <linux/kcmp.h>
+#ifdef HAVE_LINUX_KCMP_H
+#  include <linux/kcmp.h>
 static int kcmp(pid_t pid1, pid_t pid2, int type,
 		unsigned long idx1, unsigned long idx2)
 {
 	return syscall(SYS_kcmp, pid1, pid2, type, idx1, idx2);
 }
+#else
+#  ifndef KCMP_FS
+#    define KCMP_FS 0
+#  endif
+#  ifndef KCMP_VM
+#    define KCMP_VM 0
+#  endif
+#  ifndef KCMP_FILES
+#    define KCMP_FILES 0
+#  endif
+static int kcmp(pid_t pid1 __attribute__((__unused__)),
+		pid_t pid2 __attribute__((__unused__)),
+		int type __attribute__((__unused__)),
+		unsigned long idx1 __attribute__((__unused__)),
+		unsigned long idx2 __attribute__((__unused__)))
+{
+	return 1;
```

Comment:
The obvious stub to me would be this:
```c
errno = ENOSYS;
return -1;
```

Does it not work?
If so, a comment would be nice.

Reply:
Updated. Thank you.